makefile

```diff
@@ -0,0 +1,12 @@
+default:
+	cat makefile
+get_synthetic_data:
+	wget -O synthetic_data.csv https://gist.github.com/Niarfe/25100a860cbb660150a3fec87378d8ac/raw/
+square_wave: synthetic_data.csv
+	gnuplot -e "set datafile separator ','; set term png; set output 'square_wave.png'; plot 'synthetic_data.csv' using 1:2 with lines"
+fibonachi: synthetic_data.csv
+	gnuplot -e "set datafile separator ','; set term png; set output 'fibonachi.png'; plot 'synthetic_data.csv' using 1:3 with lines"
```

Comment:
Technically `clean` would be a `.PHONY`: job too, but it's pretty clear that this and the `default` are phonys. No action, just a comment.